tox.ini

```diff
@@ -29,7 +29,7 @@ commands =
 setenv =
     COVERAGE_FILE = {toxinidir}/.coverage.{envname}
 deps =
-    -r {toxinidir}/requirements_test.txt
+    -r {toxinidir}/requirements_test_min.txt
```

Comment:
Hmm, good point. Yes, `tox` fails completely. Removing `py36` from the `envlist` doesn’t really solve the goal either, which is to test that version locally, and would just make things harder on other contributors running with 3.6.2+.

I’ll take another look at these and ponder on it a bit. Maybe move the pytest extensions to _min or otherwise refactor the way things are broken down? Or add `python_requires` markers where needed (although that feels a bit like polluting the code). Or just drop this part of the change, especially if it’s not worth adding CI coverage for 3.6.0?

Reply:
`python_requires` would work. As I understand it, we just need to make sure black isn't installed for `3.6.0`, am I right?

Reply:
Sorry, missed until now. The trick with the conditional factors is that they have to appear in the env name, it couldn't be automatic based on the `python_full_version`. Closest we'd get would be something like `tox -e py36-min` that installed `requirements_test_min.txt` instead of `requirements_test.txt`. The `python_full_version` markers would let the pytest extensions run on 3.6.0 if we needed them to.

Turns out there were only three packages needing markers... let me know what you think. Happy to keep iterating on this.